### PR TITLE
Init configuration

### PR DESCRIPTION
**Pull request by chiztechnology**
This pull request contains foundations for your Bookstore website. we created the React part of the app and we also set up routing using React Router.

## Project requirements

- Initialise React app. ✅ 💯 
- The building blocks of your app should be set as re-usable components. Create a directory for your components. ✅ 💯 
- Add [React Router](https://github.com/remix-run/react-router/blob/dev/examples/basic/src/App.tsx) and set two <Route>s and <Link>s for the app's navigation: ✅ 💯 

### Books - the default view:
- Should display the list of books (empty at this point but it should be ready for the data) with the Remove button (no funcionality yet). ✅ 💯 
- Create a component called Book for displaying a single book (receive a title and an author as prop) and reuse it in a component that displays a list of books. ✅ 💯 
- Should display a form for adding a book (no functionality yet). ✅ 💯 
- Create a separate component for this form (with inputs for a title and an author [IMPORTANT in the design you can see an input for a category - please replace it with an author.]). ✅ 💯 

### Categories:
- Should display a button "Check status" only. ✅ 💯 
- Styling is not required at this point. ✅ 💯

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import './index.css';
+import App from './App';
```
Comment (MeqdamAlqudah) [inline src/index.js]: - [ ] Please remove this import index.css code since it cause errors when the user try to init the app

**Review by MeqdamAlqudah — CHANGES_REQUESTED** [review]
Hi @chiztechnology , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

image

 ## Highlights
:+1: no linter errors

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by chiztechnology** [comment]
done.
thanks dear @MeqdamAlqudah  for your advice.
I submit for another review then.

Thank you

**Review by anagudelogu — APPROVED** [review]
# STATUS: APPROVED! 🍾 🎆 🥇 

Hi @chiztechnology , 
I'm Andres.

You did a great job working on this milestone, your project is complete! 🚀 
There is nothing else to say other than... it's time to merge it 🤖  
Congratulations! 🎉 



## Highlights
✅ Great use of Git flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Descriptive PR title, summary and commit messages. 💬 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**